steps/python/python.release.checks.yml

```diff
@@ -0,0 +1,38 @@
+# python.release
+#
+# Release Python package to PyPI.
+#
+
+parameters:
+  pypiConnector: ""
+
+steps:
+- checkout: self
+
+- script: |
+    git config --global user.email "noreply@tomtom.com"
+    git config --global user.name "tt-ci"
+  displayName: "Configure git"
+
+# Checkout branch since by default Azure Pipelines do checkouts of specific commits (detached HEAD)
+# and we need to be at master to be able to push version bump changes.
+- powershell: write-host "##vso[task.setvariable variable=branch]$($env:BUILD_SOURCEBRANCH.substring($env:BUILD_SOURCEBRANCH.indexOf('/', 5) + 1))"
+
+- script: git checkout $(branch)
+  displayName: "Checkout branch"
+
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: "3.6"
+    architecture: "x64"
+
+- script: |
+    pip install --no-cache-dir \
+      bumpversion==0.5.3 \
+      twine==1.12.1 \
+      setuptools==40.8.0 \
+      pip==19.0.3
+  displayName: "Install publish prerequisites"
+
+- script: "bumpversion --dry-run --message 'Bump version: {current_version} -> {new_version} ***NO_CI***' release"
```

Comment:
Could we instead add this "check" feature to the existing release template?

Reply:
Maybe, right now I'm playing with it (provided the draft to get initial feedback like this). Once I know what all will go into it, I will revisit it.